pkg/steps/source.go

```diff
@@ -38,22 +38,40 @@ const (
 
 	ProwJobIdLabel = "prow.k8s.io/id"
 
-	gopath = "/go"
+	gopath           = "/go"
+	sshPrivateKey    = "/sshprivatekey"
+	githubSshKeyScan = "github.com,140.82.118.4,140.82.113.3,192.30.253.113 ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ=="
```

Comment:
This needs to be a configuration for `plank` and should exist outside of this PR

Reply:
opened [openshift/release/pull/5177](https://github.com/openshift/release/pull/5177)